x/ibc/02-client/keeper/client.go

```diff
@@ -69,34 +66,7 @@ func (k Keeper) UpdateClient(ctx sdk.Context, clientID string, header exported.H
 		err             error
 	)
 
```

Comment:
It's a little strange the the client keeper has `UpdateClient` and `CheckMisbehaviourAndUpdateState`. We should update these to be consistent? These funcs also do checks on the client state